### PR TITLE
CDD-1556_test_cleanup

### PR DESCRIPTION
**Pull request by paulblakehmrc**

### CODE REVIEW

File: test/unit/connectors/ExportsConnectorSpec.scala [hunk 1]
```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
...
 
-        awaitSubscriptionFields
+        awaitRequest
 
         verify(mockWsPost).POSTString(anyString, ameq(xml.toString()), any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), any[HeaderCarrier](), any[ExecutionContext])
       }
 
-      "set the content type header" in {
+      "ensure the content type header in passed through in the request" in {
```
Comment (mattclark-zerogravit) [inline test/unit/connectors/ExportsConnectorSpec.scala]: Typo? "ensure the content type header is passed through in the request"

File: test/unit/connectors/ExportsConnectorSpec.scala [hunk 2]
```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
...
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.CONTENT_TYPE -> (MimeTypes.XML + "; charset=UTF-8"))
       }
 
-      "set the accept header" in {
+      "ensure the accept header in passed through in the request" in {
```
Comment (mattclark-zerogravit) [inline test/unit/connectors/ExportsConnectorSpec.scala]: as above "ensure the accept header is...."

File: test/unit/connectors/ExportsConnectorSpec.scala [hunk 3]
```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
...
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.ACCEPT -> MimeTypes.XML)
       }
 
-      "set the date header" in {
+      "ensure the date header in passed through in the request" in {
```
Comment (mattclark-zerogravit) [inline test/unit/connectors/ExportsConnectorSpec.scala]: as above "ensure the date header is...."

File: test/unit/connectors/ExportsConnectorSpec.scala [hunk 4]
```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
...
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.DATE -> httpFormattedDate)
       }
 
-      "set the X-Forwarded-Host header" in {
+      "ensure the X-FORWARDED_HOST header in passed through in the request" in {
```
Comment (mattclark-zerogravit) [inline test/unit/connectors/ExportsConnectorSpec.scala]: as above "ensure the X-FORWARDED_HOST header is...."

File: test/unit/connectors/ExportsConnectorSpec.scala [hunk 5]
```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
...
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.X_FORWARDED_HOST -> "MDTP")
       }
 
-      "set the X-Correlation-Id header" in {
+      "ensure the X-Correlation-Id header in passed through in MDG request" in {
```
Comment (mattclark-zerogravit) [inline test/unit/connectors/ExportsConnectorSpec.scala]: as above "ensure the X-Correlation-Id header is.."

Comment (paulblakehmrc) [reply test/unit/connectors/ExportsConnectorSpec.scala]: Fixed

File: test/unit/connectors/ExportsConnectorSpec.scala (hunk 4 above)
Comment (paulblakehmrc) [reply test/unit/connectors/ExportsConnectorSpec.scala]: Fixed

File: test/unit/connectors/ExportsConnectorSpec.scala (hunk 3 above)
Comment (paulblakehmrc) [reply test/unit/connectors/ExportsConnectorSpec.scala]: Fixed

File: test/unit/connectors/ExportsConnectorSpec.scala (hunk 2 above)
Comment (paulblakehmrc) [reply test/unit/connectors/ExportsConnectorSpec.scala]: Fixed

File: test/unit/connectors/ExportsConnectorSpec.scala (hunk 1 above)
Comment (paulblakehmrc) [reply test/unit/connectors/ExportsConnectorSpec.scala]: Fixed

**Pull request merged**